### PR TITLE
added solution

### PR DESCRIPTION
**Pull request by tarkaiev**

### CODE REVIEW

File: src/main/java/core/basesyntax/Car.java [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
+        this.model = model;
+        this.color = color;
+        this.maxSpeed = maxSpeed;
+        this.engine = engine;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
+
+        return new Car(manufacturer, model, color, maxSpeed, engine);
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Car.java:21]: what do you think, will engine be cloned? what if we change engine name on the original car?

also, blank line on line 20 is redundant

File: src/main/java/core/basesyntax/Engine.java [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+    private String fuelType;
+    private int cylinderCount;
+    private int fuelConsumption;
+    private int horsePower;
 
+    public Engine(String name,
+                  String fuelType,
+                  int cylinderCount,
+                  int fuelConsumption,
+                  int horsePower) {
```
Comment (KseniiaMakarova) [inline src/main/java/core/basesyntax/Engine.java:14]: this can be placed on two lines

there is no need to wrap each field onto a new line

Comment (tarkaiev) [reply src/main/java/core/basesyntax/Engine.java:14]: fixed

File: src/main/java/core/basesyntax/Car.java (hunk 1 above)
Comment (tarkaiev) [reply src/main/java/core/basesyntax/Car.java]: tested, found mistake, thank you
fixed

**Pull request closed without merging**